AdAway/src/main/res/values-de/strings_preferences.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+
```

Comment:
No idea why `strings_preferences.xml` was changed at all... I just wanted to help with some previous untranslated strings. 

I used stringlate on android for translating. Maybe that was not a good idea. I am sorry for any inconvenience.

Reply:
@mroettgen Because the main `pref_enable_systemless_def` string was moved in here some time ago as it seems. Stringlate probably recognized the discrepancy and synced it and therefore touched this file and formatted it strangely.